pages/common/zmv.md

```diff
@@ -0,0 +1,28 @@
+# zmv
+
+> Move or rename files matching the regex pattern to corresponding files having names of the form given by destination.
+> More information: <http://zsh.sourceforge.net/Doc/Release/User-Contributions.html>.
+
+- Below renames `foo.log` to `foo.txt`, and so on:
+
+`zmv '{{(*).log}}' '{{$1.txt}}'`
+
+- Preview the command result without any changes to the files:
+
+`zmv -n '{{(*).log}}' '{{$1.txt}}'`
+
+- Interactively move files, with a prompt before every change:
+
+`zmv -i '{{(*).log}}' '{{$1.txt}}'`
+
+- Verbosely print each command as it’s being executed:
+
+`zmv -v '{{(*).log}}' '{{$1.txt}}'`
+
+- Force `cp` (copy) regardless of the name of the function. Same as `zcp`:
```

Comment:
I'm confused. What does this actually do? Same for the example below.

Reply:
`zmv` moves/renames the files deleting the old files.

If you want to `copy` the files, you can use this flag (keeps both old and new).
If you want to `create` links, you can use the flag below (keeps both old and new).

Reply:
This seems like an overloading of the tool's purpose, and muddies the waters regarding the dedicated tools that exist to perform these actions. I would suggest omitting these two examples (`-C` and `-L`) and adding a "See also" note in the main description pointing to `zcp` and `zln` instead.